src/Microsoft.AspNetCore.Mvc.Core/ActionResultOfT.cs

```diff
@@ -50,6 +50,18 @@ public ActionResult(ActionResult result)
             return new ActionResult<TValue>(result);
         }
 
+        public static implicit operator ActionResult<TValue>(ProblemDescription problemDescriptor)
```

Comment:
This shouldn't be here, this isn't the right response type for all problem descriptions